aep/general/0004/aep.md.j2

```diff
@@ -73,14 +73,19 @@ For OpenAPI 3.0, Resources must be defined in `#components/schemas` and use the
             "projects/{project}/user-events/{user-event}",
             "folder/{folder}/user-events/{user-event}",
             "users/{user}/events/{user-event}"
-          ]
+          ],
+          "parents": ["project", "folder", "user"]
         }
       }
     }
   }
 }
 ```
 
+- If the resource has any parents, they must be specified in the parents field
+using the singular name of the resource. The resource must be defined with a
+`x-aep-resource` annotation.
```

Comment:
nit: not directly related, but can you add an example of the annotation in https://aep.dev/124/#many-to-many-associations as well? we don't document the usage of that extension.